tests/test_authentication.py

```diff
@@ -134,13 +134,21 @@ def setUp(self):
         self.existing_user = User.objects.create_user(
             **self.existing_user_data)
         self.client = APIClient()
+    
+    @staticmethod
+    def validator(email):
```

Comment:
Is there a reason for moving this method into the tests?

Reply:
I saw the test fail and I know that the email isnt being verified so I tried verifying it before it was tested . It didnt work so I was to remove it from tests once I found solution on like a final commit

Reply:
let me take it out now